src/server/raftkv2/mod.rs

```diff
@@ -313,4 +317,51 @@ impl<EK: KvEngine, ER: RaftEngine> tikv_kv::Engine for RaftKv2<EK, ER> {
             }
         }
     }
+
+    fn start_flashback(
+        &self,
+        ctx: &Context,
+        start_ts: u64,
+    ) -> BoxFuture<'static, tikv_kv::Result<()>> {
+        // Send an `AdminCmdType::PrepareFlashback` to prepare the raftstore for the
+        // later flashback. Once invoked, we will update the persistent region meta and
+        // the memory state of the flashback in Peer FSM to reject all read, write
```

Comment:
> "reject all read"

Can local read still work? If yes, then will it be a problem? As from user perspective, local read and raftstore read (not exist in v2, but exist in v1) make no difference.

Reply:
Local read only works for special read requests that have FLASHBACK flag.
It won't be a problem, because it behaves like raftstore v1.